apps/viewer/package.json

```diff
@@ -44,6 +44,7 @@
     "@types/react": "18.0.26",
     "@types/sanitize-html": "2.6.2",
     "dotenv": "16.0.3",
+    "cuid": "2.1.8",
```

Comment:
Why?

Reply:
required by apps\viewer\src\features\blocks\inputs\fileUpload\fileUpload.spec.ts

Reply:
What do you think is needed?